modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/styles/components/_input.scss

```diff
@@ -34,3 +34,11 @@ label {
 	margin-right: 0;
 	width: 13px;
 }
+
+.containerPassword {
+	height: 350px !important;
+	margin-left: 27% !important;
+	padding: 20px !important;
+	padding: 50px !important;
+	width: 800px !important;
```

Comment:
@welitonpe do you have any suggestion about that?

There's 2 paddings

```
padding: 20px !important;
padding: 50px !important;
```

We should pick 1.

margin-left: 27% !important; (Using Percent) is not a good idea, because this can be complicated when you use a small screen, for example

And I noticed all the styles are using !important, is that necessary?

Reply:
The duplicate property need to be removed `padding:20px !important`.

The `margin-left: 27% !important` needs to be changed to  `margin: auto`.

Only one `!important` is needed, the important of property `width:800px !important`.